include/grpc/grpc_security.h

```diff
@@ -131,6 +131,34 @@ grpc_credentials *grpc_google_iam_credentials_create(
     const char *authorization_token, const char *authority_selector,
     void *reserved);
 
+/* Function to be called by the metadata credentials plugin implementation when
+   the metadata is ready. */
+void grpc_credentials_plugin_metadata_notify(
+    void *core_context, const grpc_metadata *creds_md, size_t num_creds_md,
```

Comment:
By calling this, core becomes the owner of metadata keys and values and caller owns the grpc_metadata array?